api/src/main/java/org/openmrs/module/fhir2/api/handler/FhirMediaComplexObsHandler.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.handler;
+
+import lombok.AccessLevel;
+import lombok.Setter;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Obs;
+import org.openmrs.Patient;
+import org.openmrs.api.APIException;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.context.Context;
+import org.openmrs.customdatatype.CustomDatatypeHandler;
+import org.openmrs.obs.ComplexData;
+import org.openmrs.obs.ComplexObsHandler;
+import org.springframework.core.Ordered;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+@Component
+@Setter(AccessLevel.PUBLIC)
+@Order(Ordered.LOWEST_PRECEDENCE)
+public class FhirMediaComplexObsHandler implements ComplexObsHandler, CustomDatatypeHandler {
```

Comment:
We really, really don't need a FHIR-specific `ComplexObsHandler` we can rely on the `ObsService` to populate the `ComplexObs` data (if any)